### DISCUSSION TITLE
Sync also other backups

### DISCUSSION THREAD

**Discussion by AFumi39** [lavinir/hassio-onedrive-backup#35] [Ideas]
I think it would be useful an option to specify which backup type you want to upload to OneDrive: only the backups generated by this component or all the backups.
For example I've scheduled a very customized backup (with or without some elements) and I'd like to archive it in OneDrive automatically. Or all the partial backups generated installing HA updates.

Thank you

**Comment by lavinir** [comment]
Added to backlog.

**Comment by lavinir** [comment]
This is now available in 1.41 (@AFumi39 )

**Reply by AFumi39** [reply]
Thank you!
